android/java/org/chromium/chrome/browser/rate/BraveRateThanksFeedbackDialog.java

```diff
@@ -0,0 +1,108 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.rate;
+
+import android.annotation.SuppressLint;
+import android.app.Dialog;
+import android.content.Context;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.Button;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
+import com.google.android.material.textfield.TextInputEditText;
+import com.google.android.material.textfield.TextInputLayout;
+
+import org.chromium.base.task.AsyncTask;
+import org.chromium.chrome.R;
+
+public class BraveRateThanksFeedbackDialog extends BottomSheetDialogFragment {
+    final public static String TAG_FRAGMENT = "brave_rate_thanks_feedback_dialog_tag";
+    private static final String TAG = "ask_play_store";
```

Comment:
Don't see where it is used, we should probably use it with logs, but name of the TAG should be different then.

Reply:
updated